simplq/src/components/pages/Admin/index.jsx

```diff
@@ -26,17 +28,27 @@ export default () => {
   const [items, setItems] = useState();
 
   const update = () => {
+    clearTimeout(timeoutId);
     if (queueId) {
       QueueService.get(queueId)
         .then((data) => {
           setItems(data.tokens);
+          timeoutId = setTimeout(update, 10000);
         })
         .catch((err) => {
           handleApiErrors(err);
+          timeoutId = setTimeout(update, 10000);
         });
     }
   };
 
+  useEffect(() => {
+    update();
+    return () => clearTimeout(timeoutId);
+  }, [queueId]);
+
+  useEffect(update, [queueId]);
```

Comment:
We don't need two? First one should take care?

Reply:
Yup let me remove the second